src/Routing.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route,
+  } from "react-router-dom";
+import PersistentDrawerLeft from './nav.js';
+import Homepage from './Homepage.js'
+import ExplorePage from './ExploreMode.js'
+import Play from './Play.js'
+import MediaCard from './Profile.js'
+import user from './nav.js'
+
+function Explore() {
+    return(
+      <div>
+        {ExplorePage()}
+      </div>
+      
+    );
+  }
+  
+  function Profile() {
+    return (
+      <div>
+        {MediaCard()}
+      </div>
+    );
+  }
+  
+  function Game(){
+    const dict = {userId : "kricheso@google.com", gridId : "board1_by_kricheso@google.com"}
```

Comment:
Next step: remove all hardcoded values

Reply:
I can't call play if I don't have that though.

Reply:
Try const dict = {userId : user.id, gridId : grid.id}. See example on how to do this in firestoremanualtests.js file. (You will need to figure out the grid part).